reviews/sandbox/2023/2023-telepresence-annual.md

```diff
@@ -0,0 +1,91 @@
+# Telepresence Sandbox Annual Review
+
+The goal of the Telepresence project is to accelerate the "developer inner loop" for cloud-native application development on Kubernetes. Telepresence achieves this by creating a dynamic network bridge between a local development environment (e.g., a laptop) and a remote Kubernetes cluster.
+
+Telepresence was presented to the TOC on April 2018. Please see the project's [Sandbox proposal](https://github.com/cncf/toc/blob/master/proposals/sandbox/telepresence.adoc).
+
+## Annual Review Questions
+
+### Metrics
+
+| Metric       |  Pre-Sandbox |     2020     |    2021      |   2022    |   2023   |
+| ------------ | ------------ | ------------ | ------------ | --------- | -------- |
+| Docker Pulls | 100K+        | 10M+         | 10M+         | 10M+      | 10M+     |
+| Stars        | 700+         | 2700+        | 4000         | 5.1k      | 5.8K     |
+| Forks        | ~50          | ~190         | ~300         | 400+      | 400+     |
+
+
+The CNCF provides Telepresence Github metrics through its [devstats site](https://telepresence.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m).
+
+### Maintainers
+
+There are eight maintainers total. Of these, four commit to the project on a daily basis and are employed by Ambassador Labs.
+The remaining four are three Ambassador Labs employees and one external maintainer, all of whom commit less frequently as needed.
+
+### Adoption
+
+The Telepresence project has seen continued growth and adoption since entering the CNCF Sandbox. In addition to the adopters mentioned last year:
+
+* Avaya
+* Bose
+* Engel & Volkers (KubeCon EU 2019 presentation: https://www.youtube.com/watch?v=tD0FIlxO1AQ)
+* InVision
+* Namely
+* PTC
+* Red Hat
+* Timecampus
+* ThetaRay
+
+We have added new adopters such as:
```

Comment:
You added these in the 2022 annual review https://github.com/cncf/toc/pull/872/files - have there been new adopters in 2023?